src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__wrap">
+          <a class="nav__link is-active" href="#">Apple</a>
+          <a class="nav__link" href="#">Samsung</a>
```

Comment:
ul should contain li elements
```suggestion
        <ul class="nav__list">
          <li class="nav__item"> 
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
          <li class="nav__item"> 
            <a class="nav__link" href="#">Samsung</a>
          </li>
```